plugin/trino-pulsar/src/test/java/io/trino/plugin/pulsar/TestPulsarConnectorTest.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.pulsar;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.testing.BaseConnectorTest;
+import io.trino.testing.MaterializedResult;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.apache.pulsar.client.admin.PulsarAdmin;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_CUSTOMER;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_LINEITEM;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_NATION;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_ORDERS;
+import static io.trino.plugin.pulsar.PulsarServer.SELECT_FROM_REGION;
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.testng.Assert.assertEquals;
+
+public class TestPulsarConnectorTest
+        extends BaseConnectorTest
+{
+    protected PulsarServer pulsarServer;
+
+    @AfterClass(alwaysRun = true)
+    public void destroy()
```

Comment:
This is not needed.

Reply:
yeah, already wrapped with closeAfterClass, removed.